### ISSUE TITLE
[Server] Writing install script for Ubuntu 12.04

### ISSUE THREAD

**Issue by saroesler** [lmm-git/Beam#16]
Special script for the server with:
- all server pacages we need (ssh, apache, my-sql, etc)
- configurations
- zikula
- zikula- modules

**Comment by cmfcmf** [comment]
I don't think that this is important. All people using this should be able to install Zikula, shouldn't they?

**Comment by lmm-git** [comment]
Of course. I think the install script should be minimized for the packages this moudle is depend on (like a ssh client, the gd-lib...). So I think it should be only a install script for this module not for the whole Zikula system. If we have to have a install script like that we should make a new preject/repository for that.
